label_studio/utils/argparser.py

```diff
@@ -39,7 +49,7 @@ def valid_filepath(filepath):
         '--force', dest='force', action='store_true',
         help='Force overwrite existing files')
     root_parser.add_argument(
-        '--root-dir', dest='root_dir', default='.',
```

Comment:
why did you remove the defaults ?

Reply:
See comment below - if we set them here, we later cannot determine whether "." was provided by the user or by default. This prevents us from determining whether "." should be overwritten by a path provided in the config file (in case it is a default) or not (in case it was provided by the user via cmd line argument)

Reply:
This is a known limit of Python's argparse and there are libraries to tackle this issue, e.g. https://pypi.org/project/ConfigArgParse/ . However, I think changing it like this is an acceptable way.